src/common/metrics/registry.js

```diff
@@ -0,0 +1,120 @@
+const WavefrontSDKCounter = require('./counter').WavefrontSDKCounter;
+const WavefrontSDKGauge = require('./gauge').WavefrontSDKGauge;
+
+/**
+ * Wavefront SDK Metrics Registry.
+ */
+class WavefrontSdkMetricsRegistry {
+  /**
+   * Construct Wavefront SDK Metrics Registry.
+   * @param wfMetricSender
+   * @param source
+   * @param tags
+   * @param prefix
+   * @param reportingIntervalSecs
+   */
+  constructor({
+    wfMetricSender,
+    source = null,
+    tags = null,
+    prefix = null,
+    reportingIntervalSecs = 60
+  }) {
+    this.wfMetricSender = wfMetricSender;
+    this.source = source;
+    this.tags = tags;
+    this.prefix = prefix ? prefix + '.' : '';
```

Comment:
prefer template strings over concatenation.

Reply:
Will make sure to change in other parts as well.